### DISCUSSION TITLE
post/homelab-journey-accessing/

### DISCUSSION THREAD

**Discussion by giscus** [ecwu/ecwu.github.io#12] [Blog Post Discussion]
# post/homelab-journey-accessing/

Homepage and Blog of ECWU

https://ecwuuuuu.com/post/homelab-journey-accessing/

**Comment by Toryelo** [comment]
已经提到了 Authentik 有比较完善的验证方法和流程。此处再次使用 zero trust 的 带保护的 application launcher 意义是什么呢。按照我的理解 完全也可以 用 Authentik 来代替。

**Reply by ecwu** [reply]
1. 有些应用并没有 SSO 的支持，这种情况下如果想要使用 Authentik 需要配合反向代理配置 forward authentication，这个配置是有些麻烦，我个人试验时一直有些小问题。
2. Cloudflare Zero Trust 我个人认为形式和 forward authentication 很像，但是配置上更简单；也允许接入自建的 Authentik（相当于 Zero Trust 做一个中间的鉴权网关）。
